tests/test_due_diligence_verification.py

```
@@ -0,0 +1,469 @@
+        # Should only show public visibility items
+        for category, data in public_data.items():
+            if isinstance(data, dict) and "documents" in data:
+                for doc in data["documents"]:
+                    assert "public" in doc.get("visibility", []), f"Non-public document in public stage: {doc['name']}"
+    
+    def test_stage_filtering_nda(self, manager):
+        """Test that NDA stage shows category completeness but hides file paths."""
+        nda_data = manager.get_stage_view("nda")
+        
+        # Should have categories but no file paths
+        for category, data in nda_data.items():
```

Ruff (0.12.2)

Loop control variable `category` not used within loop body

Rename unused `category` to `_category`

(B007)